src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -15,105 +16,97 @@ import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
 const PrepareBanner = data => {
+  getExtraKnobs(data);
+  if (data.link.link.label) {
+    data.link.link.label = text(
+      'Link label',
+      data.link.link.label,
+      buttonLabels.required
+    );
+  }
+
+  if (data.title) {
```

Comment:
@Joosthe, you duplicated this knob here. But anyway, i don't understand the reason why of the check you do for the title and the description, if am not wrong they are in all the variants no..? So you can remove the ifs for them.

Reply:
This is still in place:
` if (data.title) `
` if (data.description) `
The question about these was: why? There is no need to check for these properties since they are always set.